didact/solx-camel-k-basic/solx-camel-k-basic-alt.didact.adoc

```diff
@@ -76,35 +93,34 @@ While our integration is running in `Dev Mode`, we can modify it and see those c
 
 Change the message sent to the `.simple()` command of the Camel route in quotes to `We just changed our first Camel K integration while it was running!`. 
 
-Save the file and see what happens in the Output channel.
+Save the file and see what happens in the Apache Camel K Output channel.
 
 == Step 5: Managing our Integration
 
 We can see what integrations we currently have running in our Minikube system in the `Apache Camel K Integrations` view in the Explorer activity (Ctrl+Shift+E).
 
-* Open the `Apache Camel K Integrations` view
+* Open the `Apache Camel K Integrations` view (link:didact://?commandId=camelk.integrations.focus[Execute^])
 
 image:https://raw.githubusercontent.com/camel-tooling/vscode-camelk/master/images/camelk-integrations-view-remove-menu.jpg[Integrations view with context menu,role="right"]
 
 From here, we can:
 
-* Hover over the `Apache Camel K Integrations` view and click the `Open Apache Camel K Operator Log` button to view it.
-* Hover over the integration name to see its current state in the tooltip. 
-* Right-click on the running integration to `Remove Apache Camel K Integration` and undeploy it.
-* Right-click on the running integration to `Follow log for running Apache Camel K Integration`
-* Right-click on the running integration to `Follow kit builder log for running Apache Camel K integration`
+* Hover over the `Apache Camel K Integrations` view and click the `Open Apache Camel K Operator Log` button to view it. (link:didact://?commandId=camelk.integrations.openOperatorLog[Execute^])
+* Hover over the integration name to see its current state in the tooltip. (link:didact://?commandId=camelk.integrations.selectFirstNode[Execute to select first integration in tree^])
+* Right-click on the running integration to `Remove Apache Camel K Integration` and undeploy it. (link:didact://?commandId=camelk.integrations.remove[DANGER! WILL REMOVE RUNNING INTEGRATION])
+* Right-click on the running integration to `Follow kit builder log for running Apache Camel K integration` (link:didact://?commandId=camelk.integrations.kitlog[Execute to view log for first integration in tree^])
```

Comment:
one missing capital letter "Follow kit builder log for running Apache Camel K Integration"